src/database/schema.ts

```diff
@@ -75,6 +75,8 @@ export type PartialRoundRole = Updateable<RoundRoleTable>;
 export type ProjectTable = {
   id: string;
   name: string;
+  nonce: bigint | null;
+  anchor: Address | null;
```

Comment:
```suggestion
  anchorAddress: Address | null;
```

it has to be modified everywhere else 👀